cocos/asset/asset-manager/asset-manager.ts

```diff
@@ -29,20 +29,16 @@ import { cclegacy } from '@base/global';
 import { Asset } from '../assets/asset';
 import { sys, Settings, settings, path, EventTarget } from '../../core';
 import Bundle from './bundle';
-import Cache from './cache';
-import { ICache } from './cache';
+import Cache, { ICache } from './cache';
 import CacheManager from './cache-manager';
-import dependUtil from './depend-util';
-import { DependUtil } from './depend-util';
-import downloader from './downloader';
-import { Downloader } from './downloader';
+import dependUtil, { DependUtil } from './depend-util';
+import downloader, { Downloader } from './downloader';
```

Comment:
as it's discussed before https://github.com/cocos/cocos-engine/pull/16247#discussion_r1324651600

we prefer to separate the default import and named import, but currently we have no suitable eslint rule to supporting to do this. 
so I revert this change to adapt to current eslint rules